### PR TITLE
Geometry pa connectivity section 3

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: src/pages/Search.jsx [hunk 1]
```diff
@@ -767,6 +807,7 @@ class Search extends Component {
       'forestIntegrity',
       'currentPAConn',
       'timelinePAConn',
+      'currentSEPAConn',
```
Comment (erikasv) [inline src/pages/Search.jsx:810]: you're missing cleaning the strategic ecosystems layers

File: src/utils/colorPalettes.js [hunk 2]
```diff
@@ -86,6 +86,9 @@ export default {
   paramo: ['#eaa04a'],
   dryForest: ['#8cb64c'],
   wetland: ['#6794cb'],
+  paramoPAConn: ['#120000'],
+  dryForestPAConn: ['#120000'],
+  wetlandPAConn: ['#120000'],
```
Comment (erikasv) [inline src/utils/colorPalettes.js:91]: these can be grouped into one palette since they are the same

File: src/utils/matchColor.js [hunk 3]
```diff
@@ -65,6 +65,18 @@ const match = {
...
+    sort: ['paramoPAConn'],
+  },
+  dryForestPAConn: {
+    palette: 'dryForestPAConn',
+    sort: ['dryForestPAConn'],
+  },
+  wetlandPAConn: {
+    palette: 'wetlandPAConn',
+    sort: ['wetlandPAConn'],
+  },
```
Comment (erikasv) [inline src/utils/matchColor.js:79]: In all these cases you don't need to define a sort since you are using the default behavior.

**Review by erikasv — CHANGES_REQUESTED** [review]

File: src/utils/matchColor.js (hunk 3 above)
Comment (cazapatamar) [reply src/utils/matchColor.js]: Done

File: src/utils/colorPalettes.js (hunk 2 above)
Comment (cazapatamar) [reply src/utils/colorPalettes.js]: Done

File: src/pages/Search.jsx (hunk 1 above)
Comment (cazapatamar) [reply src/pages/Search.jsx:810]: Done

**Review by erikasv — APPROVED** [review]

**Review by Trjegul84 — APPROVED** [review]

**Pull request merged**